spec/requests/api_new/users_controller_request_spec.rb

```diff
@@ -3,90 +3,103 @@
 
 describe ApiNew::UsersController, type: :request do
 
-
-	context 'for unlogged in user' do
-		it 'returns unauthorized when not logged in' do
-			get "/api_new/users/current"
-			expect(response).to have_http_status(:unauthorized)
+	context 'GET /api_new/users/current' do
```

Comment:
Please undo all of the changes that don't relate to the spec you're changing. It makes reviewing pretty hard